Disable testOpenAndCloseWebsocket JSF 2.3 FAT for EE10 until we debug failures

> [09/27/2022 15:44:35:304 PDT] 001 JSF23WebSocketTests            testOpenAndCloseWebsocket_EE10 I Looking for 'Called onclose listener' in string: JSF 2.3 WebSocket test
> JSF 2.3 WebSocket - Test that onopen and onclose listener can be triggered manually, that is, when connected attribute is set to false 
>  Click here to open channelClick here to close channel
> 
> Called onopen listener
> [09/27/2022 15:44:35:307 PDT] 001 FATRunner                      evaluate                       S null
> java.lang.AssertionError: Expected to find 'Called onclose listener' in response, but response was: JSF 2.3 WebSocket test

`Called onopen listener` worked , but not `Called onclose listener`

WebsocketPushBean logging:

> [9/27/22, 15:43:49:400 PDT] 0000002e com.ibm.ws.jsf23.fat.websocket.WebsocketPushBean             I Channel myChannel was opened successfully!
> [9/27/22, 15:43:49:766 PDT] 0000002e com.ibm.ws.jsf23.fat.websocket.WebsocketPushBean             I Channel myChannel was closed successfully!
> [9/27/22, 15:44:05:115 PDT] 0000002b com.ibm.ws.jsf23.fat.websocket.WebsocketPushBean             I Channel myChannel was opened successfully!
> [9/27/22, 15:44:05:288 PDT] 0000002c com.ibm.ws.jsf23.fat.websocket.WebsocketPushBean             I Channel myChannel was closed successfully!

--- a/dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23WebSocketTests.java
+++ b/dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23WebSocketTests.java
@@ -35,8 +35,8 @@ import com.ibm.ws.jsf23.fat.JSFUtils;
 import componenttest.annotation.Server;
 import componenttest.annotation.SkipForRepeat;
 import componenttest.custom.junit.runner.FATRunner;
-import componenttest.topology.impl.LibertyServer;
 import componenttest.rules.repeater.JakartaEE10Action;
+import componenttest.topology.impl.LibertyServer;
 
 /**
  * This test class is to be used for the tests that test feature specified
@@ -91,7 +91,7 @@ public class JSF23WebSocketTests {
             String contextRoot = "WebSocket";
             URL url;
 
-            if(JakartaEE10Action.isActive()){
+            if (JakartaEE10Action.isActive()) {
                 url = JSFUtils.createHttpUrl(server, contextRoot, "faces40/PushWebSocketTest.jsf");
             } else {
                 url = JSFUtils.createHttpUrl(server, contextRoot, "PushWebSocketTest.jsf");
@@ -145,6 +145,7 @@ public class JSF23WebSocketTests {
      * @throws Exception
      */
     @Test
+    @SkipForRepeat(SkipForRepeat.EE10_FEATURES)
     public void testOpenAndCloseWebsocket() throws Exception {
         try (WebClient webClient = new WebClient()) {
             webClient.getOptions().setThrowExceptionOnScriptError(false);
@@ -153,7 +154,7 @@ public class JSF23WebSocketTests {
             String contextRoot = "WebSocket";
             URL url;
 
-            if(JakartaEE10Action.isActive()){
+            if (JakartaEE10Action.isActive()) {
                 url = JSFUtils.createHttpUrl(server, contextRoot, "faces40/OpenCloseWebSocketTest.jsf");
             } else {
                 url = JSFUtils.createHttpUrl(server, contextRoot, "OpenCloseWebSocketTest.jsf");

